src/wp-admin/includes/user.php

```diff
@@ -660,3 +660,27 @@ function wp_is_authorize_application_password_request_valid( $request, $user ) {
 
 	return true;
 }
+
+/**
+ * Is this site protected by HTTP Basic Auth.
+ *
+ * This is used to disable creating Application Passwords since it uses Basic Auth under the hood.
+ *
+ * @since 5.6.1
+ *
+ * @return bool
+ */
+function wp_is_site_protected_by_basic_auth() {
```

Comment:
A better place for this new function seems to be wp-includes/load.php.

Reply:
👍 will move it there.